bin/uglifyjs

```diff
@@ -121,6 +120,11 @@ if (program.parse) {
         fatal("ERROR: inline source map only works with built-in parser");
     }
 }
+if (~program.rawArgs.indexOf("--rename")) {
```

Comment:
Since rawArgs is undocumented should be:
```js
if (program.rawArgs && ~program.rawArgs.indexOf("--rename")) {
```
unless you want to lock down a specific version of `commander` in `package.json`.

Reply:
I think `rawArgs` is the least of your worries - try swapping `--rename` and `--no-rename` above and see the tests fail.

So the question is whether to have CLI throws an error, or silently disables `rename` when `commander` makes a radical change.

Reply:
I think we'll lock down `commander` version _if_ those unit tests started failing.

Reply:
Would Travis necessarily pull in the latest version of `commander` matching the `package.json` pattern or just use whatever it had cached?

Reply:
It doesn't seem to keep `npm`'s cache, and we certainly haven't configured the build cache to save those files.